### DISCUSSION TITLE
TIMEOUT "answers" logged as CLIENT_QUERY

### DISCUSSION THREAD

**Discussion by flz** [dmachard/DNS-collector#1225] [General]
I realize that TIMEOUT are not really answers but would it make more sense to log them as CLIENT_RESPONSE than CLIENT_QUERY?

**Comment by dmachard** [comment]
I should also make changes to timestamp (arrival time of the query + timeout)
Makes sense if I do that.

**Comment by flz** [comment]
The complexity here is that if you're receiving the response after the timeout has been reached, you will end up with more CLIENT_RESPONSE than CLIENT_QUERY, which isn't right either.

**Comment by dmachard** [comment]
You're right, 
- perhaps we can add a new setting to let the choice of CLIENT_RESPONSE or CLIENT_QUERY
- for your last point, If you have some suggestions :)
